t/obs/openSUSE:Factory:ARM:ToTest/jeos-armv7hl/print_openqa.before

```diff
@@ -36,13 +36,13 @@
  MIRROR_HTTPS=https://openqa.opensuse.org/assets/repo/openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610 \
  MIRROR_PREFIX=http://openqa.opensuse.org/assets/repo \
  REPO_0=openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610 \
- REPO_1=openSUSE-Tumbleweed-oss-armv7hl.license-Snapshot20240610 \
- REPO_2=openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610-debuginfo \
- REPO_3=openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610-source \
+ REPO_1=openSUSE-Tumbleweed-oss-armv7hl-Snapshot20240610-Debug \
```

Comment:
Is there really a repo *-armv7hl-* being synced? It's one large multi-arch repo now; and from the rsync commands, I don't think this path would be valid

Reply:
There shouldn't, the generation of the files is kinda wrong